doc/sphinxext/gallery_generator.py

```diff
@@ -24,7 +24,7 @@
 
 from bokeh.io import export_png
 from bokeh.layouts import gridplot
-
+from arviz.rcparams import rc_context
```

Comment:
I can't seem to import rcparam file and am getting the error `ValueError: attempted relative import beyond top-level package`. Is there any workaround to this?

Reply:
Do either of `import arviz` and then use `arviz.rc_context` or `from arviz import rc_context` work?

Reply:
Sorry for the miscommunication, actually `from ....arviz/rcparams import rc_context` was showing the error. `from arviz.rcparams import rc_context` is working fine.